rest/codegen/paths.go

```diff
@@ -52,17 +93,20 @@ type (
 
 var pathTmpl = template.Must(template.New("path").
 	Funcs(template.FuncMap{
-		"join": strings.Join,
-		"add":  codegen.Add,
+		"add":       codegen.Add,
+		"goTypeRef": codegen.GoTypeRef,
 	}).
 	Parse(pathT))
 
 // PathWriter returns the path generators writer.
-func PathWriter(r *design.RootExpr) codegen.FileWriter {
+func PathWriter(api *goadesign.APIExpr, r *design.RootExpr) codegen.FileWriter {
+	title := fmt.Sprintf("%s HTTP request path constructors", api.Name)
 	sections := []*codegen.Section{
-		codegen.Header("", "http", []*codegen.ImportSpec{
-			{Path: "context"},
+		codegen.Header(title, "http", []*codegen.ImportSpec{
```

Comment:
Are there any plans to stripout unused imports after generation? 
For example running `goimports` on every file after generation.

Because i include all the imports i think we would need. And in some cases not all the imports are needed. 
The scenarios where this will break at compile time:
- Simple paths with no params (no import needed)
- Paths with non sliced params (fmt needed)
- Paths with slices of string or interface (fmt, net/url, strings)
- Paths with slices of numbers or boolean (fmt, strconv, strings)

If there are no plans, we need to figure out a way to determine the required imports.